07_payment_channels.asciidoc

```diff
@@ -68,7 +68,7 @@ Let's assume that Alice wants to construct a payment channel allowing her to con
 Every node on the Lightning Network is identified by a _node public key_. The public key uniquely identifies the specific node and is usually presented as a hexadecimal encoding. For example, Rene Pickhardt currently runs a Lightning Node (+ln.rene-pickhardt.de+) that is identified by the following node public key:
 
 ----
-03efccf2c383d7bf340da9a3f02e2c23104a0e4fe8ac1a880c8e2dc92fbdacd9df
+02a1cebfacb2674143b5ad0df3c22c609e935f7bc0ebe801f37b8e9023d45ea7b8
```

Comment:
I am fine with having my public node id there it's connected to my identity anyway